docs/ARDUPILOT.md

```diff
@@ -23,19 +23,59 @@ For my Matek H743 board the configuration is:
 - SERIAL7_OPTIONS = 0
 - SERIAL7_PROTOCOL = 23
 
-Notes & References:
+***Notes & References***:
 - [ArduPilot Docs for CRSF](https://ardupilot.org/copter/docs/common-tbs-rc.html)
 - [ArduPilot Docs for RC_PROTOCOLS](https://ardupilot.org/plane/docs/parameters.html#rc-protocols-rc-protocols-enabled)
 - [ArduPilot Docs for RSSI_TYPE](https://ardupilot.org/plane/docs/parameters.html#rssi-type-rssi-type)
-- There are more RC options available in the 'RC_OPTIONS' parameter. ([ArduPilot Docs for RC_OPTIONS](https://ardupilot.org/plane/docs/parameters.html#rc-options-rc-options)) 
+- There are more RC options available in the 'RC_OPTIONS' parameter. ([ArduPilot Docs for RC_OPTIONS](https://ardupilot.org/plane/docs/parameters.html#rc-options-rc-options))
+
+## RSSI
+
+RSSI is a relatively confusing topic with ArduPilot, since there are so many ways.
```

Comment:
Perhaps 'RSSI is a relatively confusing topic with ArduPilot, since there are multiple metrics and sources.'

Reply:
thx. done